### PR TITLE
Clarify request mode

### PR DESCRIPTION
**Pull request by rogaldh**

### CODE REVIEW

File: src/account.js [hunk 1]
```diff
@@ -43,7 +43,10 @@ export default class Account<Config: AccountConfig, Storage: AbstractStorage> {
       retries: config.retries || MAX_AJAX_RETRY,
     }
     this.leeway = config.leeway || LEEWAY
-    this.requestMode = config.requestMode || 'id'
+    this.requestMode = config.requestMode || 'me'
+
+    // eslint-disable-next-line no-param-reassign
+    if (this.requestMode === 'me') config.label = 'me'
```
Comment (manifest) [inline src/account.js]: `me` should never be a label

File: src/account.js [hunk 2]
```diff
@@ -54,21 +57,22 @@ export default class Account<Config: AccountConfig, Storage: AbstractStorage> {
...
   // eslint-disable-next-line class-methods-use-this
-  _createId (audience: string, label: string = 'me', separator: string = '.'): { label: string, id: string } {
+  _createId (audience: string, label: string, separator: string = '.'): { label: string, id: string } {
     if (!audience) throw new TypeError('`audience` is absent')
+    if (!label) throw new TypeError('`label` is absent')
 
     return {
-      label,
-      id: `${label}${separator}${audience}`,
+      id: label,
```
Comment (manifest) [inline src/account.js]: `label` here is `account_label`. Why do we call it id?

File: src/account.js [hunk 3]
```diff
@@ -54,21 +57,22 @@ export default class Account<Config: AccountConfig, Storage: AbstractStorage> {
...
+      label: `${label}${separator}${audience}`,
```
Comment (manifest) [inline src/account.js]: `account_label + audience` is `account_id`. Why do we call it `label`?

File: src/account.js (hunk 1 above)
Comment (rogaldh) [reply src/account.js]: Fixed

File: src/account.js (hunk 3 above)
Comment (rogaldh) [reply src/account.js]: Fixed

File: src/account.js (hunk 2 above)
Comment (rogaldh) [reply src/account.js]: Fixed

**Pull request merged**